package.json

```diff
@@ -35,6 +35,7 @@
 	"engines": {
 		"node": "20.x"
 	},
+	"packageManager": "pnpm@9.1.0+sha256.22e36fba7f4880ecf749a5ca128b8435da085ecd49575e7fb9e64d6bf4fad394",
```

Comment:
@edmundhung this field allows us to use `corepack` so now in CI or development, you can just run `corepack enable` and it'll automatically use the correct package manager.

to update you can run `corepack up` and that will also rewrite the hash for the version number like it is here.

Reply:
Do you recommend corepack? I think I have tried it once but there were some issue with pnpm so I didnt use it at the end. If you suggest this, I can give it another try.

Reply:
@edmundhung I use it on my project at work, I'd love to hear about the issues you had, with the setup I've done here, I haven't personally witnessed any issues.